### ISSUE TITLE
set seed in weight init

### ISSUE THREAD

**Issue by varghesealex90** [alykhantejani/nninit#4]
Hi ;

I am implementing a fully conv network which has conv and transposed convolution modules in them. I was wondering if it possible to set the seed so that I would be able to reproduce my results . Below is the code I wrote for weight init.

def init_weights(m):
    if type(m) == nn.Conv2d :
        init.xavier_uniform(m.weight, gain=np.sqrt(2))
    if type(m)==nn.ConvTranspose2d:
        init.xavier_uniform(m.weight, gain=np.sqrt(2))

model.apply(init_weight)

**Comment by alykhantejani** [comment]
Hi @varghesealex90 

Just a side note, this module is now integrated into the latest pytorch release and you should use [that version](http://pytorch.org/docs/master/nn.html#torch-nn-init) (as that is the one being maintained). 

However, you should be able to set the seed by setting the randomstate/seed in `random` and in `torch`. 

Closing the issue here, if you run into any other issues please open an issue on the PyTorch repo.
